packages/pharos/src/components/sheet/pharos-sheet.ts

```diff
@@ -73,6 +73,27 @@ export class PharosSheet extends ScopedRegistryMixin(PharosElement) {
   @property({ type: String, reflect: true })
   public header = 'Sheet header';
 
+  /**
+   * Start height for omitOverlay mode, only accepts px (e.g. "200px")
+   * @attr start-height
+   */
+  @property({ type: String, attribute: 'start-height', reflect: true })
+  public startHeight = '60px';
```

Comment:
How do we feel about using pharos-spacing-tokens to define these pixel values instead?

I was thinking about users of the sheet being able to pass a number divisible by 0.25 and then that getting converted over to pharos-spacing-tokens.